README.md

```diff
@@ -1,31 +1,100 @@
+
+### Maintainers
+The Maintainers of the project are the people that are responsible for the project.
+Each project should have 1-3 Maintainers.
+A Maintainer should drive the project's roadmap/vision and take care of topics like community meetings,
+the general structure of the project (fits our best practices, is secure, ...), and so on.
+In addition, the Maintainers should report project updates to the Hiero **Technical Steering Committee** (TSC).
+A Committer of a project can be nominated to become a Maintainer of that project, and all Maintainers should vote on that.
+The vote should be held in a GitHub PR (by doing a change to the `config.yaml` file).
+It is not the goal that every Committer of a project becomes a Maintainer.
+Having hundreds of Maintainer for a single project does not make sense.
+As already defined for the Committer role, a Maintainer can lose the Maintainer role if they harm the project in any way.
+For a Maintainer the contribution to the project should be on a really high level.
+Therefore, it is a valid decision to remove a Maintainer from the project and give that person the Committer role after a period of inactivity.
+Removing a Maintainer from the project and giving that person the Contributor role should only happen in
+really extreme cases where the Maintainer is harming the project or ignoring the Code of Conduct.
 
 ## Definition of groups
 
-The `config.yaml` file defines several groups that all have different usecases:
+Since Hiero contains multiple projects, we need to define GitHub **groups** that are used to manage the access to the GitHub repositories.
+The groups are defined in the `config.yaml` file and are used to manage the access to the GitHub repositories that belong to a project.
+Contributors do not have any specific rights, and therefore we do not need to define any groups for them.
+Committers and Maintainers are defined in groups that are used to manage the access to the GitHub repositories.
+For each project we have defined a Committers group and a Maintainers group: 
 
+- **`PROJECT-maintainers`**: For each project we have a group that defines the Maintainers of that project.
+- **`PROJECT-committers`**: For each project we have a group that defines the Committers of that project.
+
+Next to that you can find some other general groups in the `config.yaml` file:
+
+- **`hiero-automation`**: CI/CD Automation service account for hiero-ledger projects
+- **`hiero-mirror-node-automation`**: CI/CD Automation serice account for hiero-ledger mirror node projects
 - **`github-maintainers`**: The group is responsible for maintaining our GitHub environment and infrastructure (including GitHub Action runners, ...).
-- **`github-committers`**:
 - **`community`**: A group that contains all people that are currently not part of any other (non-temporary) group.
-  We use this temporarly to invite people to the Hiero org.
```

Comment:
The community group is gonna be removed:
https://github.com/hiero-ledger/governance/pull/215